package.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "worker-nodes",
+    "version": "2.5.0-alpha",
```

Comment:
remove this version before releasing new one ;)

Reply:
I'm running that on our project's CI, so I'm installing it via yarn like so:
```
yarn add worker-nodes@yurynix/node-worker-nodes#8010dc0fbc322c2022a057f758fa9207363ebceb
```
and then pushing to github for the CI to run.

`yarn add` from github fails if there is no `version` field unfortunately, due `npm pack` failing behind the scenes.

Reply:
ok, I'll think how to handle this